### PR TITLE
Task3

### PR DESCRIPTION
**Pull request by Popov-Maxim**
when click on the rating, it will jump to the review page. 
I didnot copy the post 10 times. 4 posts to look at styles. last post has comments and links to delete and edit

### CODE REVIEW

File: UI/index.html [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+					выйти
+				</div>
+			</div>
+		</header>
+		<main>
+			<div class="filter-table">
+				<div class="filter-tools">
+					<div class="filter-tools__date">
+						Дата:
+						<input type="text" class="input-date" style="border-radius: 20px 0 0 20px" placeholder="с...">
```
Comment (YauheniyaBasaranovich) [inline UI/index.html]: Please don't use inline styles for no reason. There are a lot of problems inline styles may cause, just a couple of them:
- it's hard to maintain: you always need to remember where and why you added some rules. And what if half of year have passed? And what if your project has 3k lines of HTML code?
- it forces you to copy-paste the same styles for billion times
- it overrides all the external and internal stylesheets you have (https://www.w3schools.com/css/css_howto.asp)

File: UI/index.html [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+							</div>
+						</div>
+						<div class="post-info__content">
+							<div class="post-info__photo">
+								<img src="photo.jpg" width="100%">
+								
+							</div>
+							<div class="post-info__description">
+								<p class="title">Краткое описание:</p>
+								<p>......................................................................</p>
```
Comment (YauheniyaBasaranovich) [inline UI/index.html]: What about 'Lorem ipsum' usage?

File: UI/index.html [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+							</div>
+						</div>
+						<div class="post-info__discount">
+									до 01.01.2021<br>
+									Скидка:20%
+						</div>
+						<div class="post-info__tags">
+							<p style="margin: 0;">Теги: </p>
+							<button class="button-tag" disabled>Тег1</button>
```
Comment (YauheniyaBasaranovich) [inline UI/index.html]: If element is not supposed to be clickable it's wrong to use `button` tag. Use `span` instead.

File: UI/index.html [hunk 4]
```diff
@@ -0,0 +1,338 @@
...
+							</div>
+							<div class="change">
+								<a href="" class="change">Редактировать</a>
+								<a href="" class="change">Удалить</a>
+							</div>
+							
+						</div>
+						<div class="post-info__author">
+							<u><p>ссылка на полное описание: <a href="https://en.wikipedia.org/wiki/Psychology" target="_blank">https://en.wikipedia.org/wiki/Psychology</a></p></u>
+							<p align="right" style="margin-top: 0;">Вендор:<br>Попов Максим</p>
```
Comment (YauheniyaBasaranovich) [inline UI/index.html]: `align` attribute is deprecated, please use styles (not inline 🙂 ) instead.

File: UI/index.html [hunk 5]
```diff
@@ -0,0 +1,338 @@
...
+									<div class="user-comment__rating">
+										<input type="radio" id="star-5" name="rating-user-2" value="5" disabled>
+										<label for="star-5" title="Оценка «5»"></label>	
+										<input type="radio" id="star-4" name="rating-user-2" value="4" disabled >
+										<label for="star-4" title="Оценка «4»"></label>    
+										<input type="radio" id="star-3" name="rating-user-2" value="3" disabled checked>
+										<label for="star-3" title="Оценка «3»"></label>  
+										<input type="radio" id="star-2" name="rating-user-2" value="2" disabled >
+										<label for="star-2" title="Оценка «2»"></label>    
+										<input type="radio" id="star-1" name="rating-user-2" value="1" disabled>
```
Comment (YauheniyaBasaranovich) [inline UI/index.html]: You have several `id="star-1"` on the page, that's a problem. 🤔

File: UI/styles.css [hunk 6]
```diff
@@ -0,0 +1,354 @@
...
+	background-color: black;
+	display: flex;
+	justify-content: space-between;
+	height: 90px;
+	align-items: center;
+	position: fixed; 
+	top: 0;
+	width: 100%;
+}
+	.header__logo {
```
Comment (YauheniyaBasaranovich) [inline UI/styles.css]: Please align code properly.

File: UI/styles.css [hunk 7]
```diff
@@ -0,0 +1,354 @@
...
+		color: white;
+		padding: 0 2%;
+		font-size: 60px;
+		font-family: "Myriad Pro",sans-serif;
+
+	}
+	.header__user-menu {
+		display: flex;
+		/*align-items: flex-end;*/
```
Comment (YauheniyaBasaranovich) [inline UI/styles.css]: If you don't need some style rules, please remove them to keep code clean.

File: UI/styles.css [hunk 8]
```diff
@@ -0,0 +1,354 @@
...
+		font-family: "Verdana";
+		
+	}
+		.filter-tools__date {
+			padding: 10px 0;
+			display: flex;
+			justify-content: space-between;
+			align-items: center;
+		}
+			.input-date{
```
Comment (YauheniyaBasaranovich) [inline UI/styles.css]: Your code editor most likely has some code formatting features. Please google them and use when you write code.

File: UI/index.html [hunk 9]
```diff
@@ -0,0 +1,338 @@
...
+				Profit!
+			</div>
+			<div class="header__user-menu">
+				<div class="header__button-add">
+					<button class="button-add">Добавить</button>
+				</div> 
+				<div class="header__name">
+					Попов Максим
+				</div>
+				<div class="header__log-out">
```
Comment (YauheniyaBasaranovich) [inline UI/index.html]: It should be button

File: UI/index.html [hunk 10]
```diff
@@ -0,0 +1,338 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Profit!</title>
+		<link rel="stylesheet" type="text/css" href="styles.css">
+	</head>
+	<body>
+		<header>
+			<div class="header__logo">
```
Comment (YauheniyaBasaranovich) [inline UI/index.html:9]: The most common way to write classes and ids in HTML is snake-case, let's use it (in this case here should be `header-logo` class).

**Review by YauheniyaBasaranovich — COMMENTED** [review]
**Please refactor your css-code!**

Rating field is a bit off on small screens:
Screen Shot 2021-03-01 at 10 07 29 PM

File: UI/index.html (hunk 10 above)
Comment (Popov-Maxim) [reply UI/index.html:9]: just I read about "block__element" way.
this method cannot be used?

File: UI/styles.css (hunk 8 above)
Comment (Popov-Maxim) [reply UI/styles.css]: If you're talking about tabs. I used this to make it easier to navigate the structure of the classes. 
Can I do that?

File: UI/index.html (hunk 3 above)
Comment (Popov-Maxim) [reply UI/index.html]: In the future I wanted to click on the tag showing all posts for this tag

File: UI/index.html (hunk 10 above)
Comment (YauheniyaBasaranovich) [reply UI/index.html:9]: It can, but why did you decide to use this particular approach?

File: UI/styles.css (hunk 8 above)
Comment (YauheniyaBasaranovich) [reply UI/styles.css]: It makes sense in case when you use preprocessors, for plain css looks like a mess.

File: UI/index.html (hunk 10 above)
Comment (Popov-Maxim) [reply UI/index.html:9]: Because I decided this is a enough intuitive way to navigate in my classes. Easier to write for me

**Comment by Popov-Maxim** [comment]
Can I merge this pull request?
It is not approved by you

**Pull request merged**